app/main.py

```diff
@@ -1,3 +1,19 @@
 def cache(func):
-    # Write your code here
-    pass
+
+    my_memory = {}
```

Comment:
Please, rename the variable
Use a descriptive name, this dictionary isn't actually your memory)